<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerieTypesComboBoxesWidget</class>
 <widget class="QWidget" name="SerieTypesComboBoxesWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>44</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QCheckBox" name="checkBox_competitions">
     <property name="styleSheet">
      <string notr="true">background: transparent; color: black;</string>
     </property>
     <property name="text">
      <string>Konkursy</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="checkBox_trialRounds">
     <property name="styleSheet">
      <string notr="true">background: transparent; color: black;</string>
     </property>
     <property name="text">
      <string>Serie pr√≥bne</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkBox_qualifications">
     <property name="styleSheet">
      <string notr="true">background: transparent; color: black;</string>
     </property>
     <property name="text">
      <string>Kwalifikacje</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_trainings">
     <property name="styleSheet">
      <string notr="true">background: transparent; color: black;</string>
     </property>
     <property name="text">
      <string>Treningi</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
